File: nodejs-backend/employee.js
class Employee {
    constructor(row) {
        this.row = row;
    }

    getRecord = () => {
        const parsedContent = this.row.split(",");
        return {
            id: parsedContent[0],
            first_name: parsedContent[1],
            last_name: parsedContent[2],
            email: parsedContent[3],
            gender: parsedContent[4],
            company: parsedContent[5],
            job_title: parsedContent[6],
        }
    }
}

module.exports = {
    Employee
};

File: go-backend/model/employee.go
package model

type Employee struct {
	Id        int    `json:"id" csv:"id"`
	FirstName string `json:"first_name"  csv:"first_name"`
	LastName  string `json:"last_name"  csv:"last_name"`
	Email     string `json:"email"  csv:"email"`
	Gender    string `json:"gender"  csv:"gender"`
	Company   string `json:"company"  csv:"company"`
	JobTitle  string `json:"job_title"  csv:"job_title"`
}
